File: examples/modulith-clinical-tool-jpa/src/main/java/io/zenwave360/example/clinicaltool/modules/clinical/core/inbound/dtos/PatientHospital.java
package io.zenwave360.example.clinicaltool.modules.clinical.core.inbound.dtos;

import io.zenwave360.example.clinicaltool.modules.clinical.core.domain.*;
import java.io.Serializable;
import java.math.*;
import java.time.*;
import java.util.*;
import jakarta.validation.constraints.*;

/**
 * PatientHospital.
 */
@lombok.Getter @lombok.Setter
public  class PatientHospital  implements Serializable {


    
    private Long patientId ;

    
    private Long hospitalId ;

    
    private String hisNumber ;

    
    private String fullName ;

    
    private GenderType gender ;

    
    private String dni ;

    
    private LocalDate birthDate ;

    
    private String phone ;

    
    private String email ;

    
    private String insuranceCardNumber ;

    
    private String address ;














}

File: examples/modulith-klinical-tool-jpa/src/main/kotlin/io/zenwave360/example/clinicaltool/modules/clinical/core/inbound/dtos/PatientHospital.kt
package io.zenwave360.example.clinicaltool.modules.clinical.core.inbound.dtos

import io.zenwave360.example.clinicaltool.modules.clinical.core.domain.*
import java.io.Serializable
import java.math.*
import java.time.*
import jakarta.validation.constraints.*

/**
 * PatientHospital.
 */
data class PatientHospital(

    
    val patientId: Long?  = null,

    
    val hospitalId: Long?  = null,

    
    val hisNumber: String?  = null,

    
    val fullName: String?  = null,

    
    val gender: GenderType?  = null,

    
    val dni: String?  = null,

    
    val birthDate: LocalDate?  = null,

    
    val phone: String?  = null,

    
    val email: String?  = null,

    
    val insuranceCardNumber: String?  = null,

    
    val address: String?  = null

)  : Serializable {













}
